cnxeasybake/oven.py

```diff
@@ -1334,3 +1340,15 @@ def copy_w_id_suffix(elem, suffix="_copy"):
     for id_elem in mycopy.xpath('//*[@id]'):
         id_elem.set('id', id_elem.get('id') + suffix)
     return mycopy
+
+
+def fetch_nodes(url):
+    """Get the contents at url, return parsed nodes child of body tag."""
+    req = requests.get(url)
+    if req.ok:
+        html = etree.HTML(req.text)
+        nodes = html.xpath('body/*')
```

Comment:
Hmm, those sort of go together: extract `#this_is_an_id` as the selector part. Adding a selector would be pretty straightforward, since I already have code for that in the `sortby/groupby` feature. I'd estimate 4 hours.